calour/heatmap/heatmap.py

```diff
@@ -124,7 +124,8 @@ def heatmap(exp, sample_field=None, feature_field=False, yticklabels_max=100,
         passed to ``norm`` parameter of ``plt.imshow``. Default is log scale.
     cax : None or matplotlib Axes object (optional)
         plot a legend colorbar for the heatmap in the cax or not
-
+    figsize : tuple of int, optional
+        Specifies the height and the width of the figure
```

Comment:
where is this `figsize` defined or used?

Reply:
I added it ... and forgot to delete it :3